frontend/src/layout/navigation/TopBar/TopBar.scss

```diff
@@ -75,6 +74,17 @@
     a:visited {
         color: var(--purple-light);
     }
+    .bold {
```

Comment:
Nit: Didn't see a change that uses this class, if possible using `strong` or `b` seems like a better approach too